platform/files_test.go

```diff
@@ -154,4 +154,80 @@ func testFiles(t *testing.T, when spec.G, it spec.S) {
 			})
 		})
 	})
+	when("analyzed.toml", func() {
+		when("it is old it stays old", func() {
+			it("serializes and deserializes", func() {
```

Comment:
Though it's painful I think we do need to compare to a hard-coded string as the style of this test is hiding the regression mentioned here: https://github.com/buildpacks/lifecycle/pull/994#discussion_r1126643263

Reply:
Update: actually there is no regression, though testing the string would be the proof!

Reply:
(we said i should test the string)